src/main/java/com/conveyal/gtfs/validator/PatternFinderValidator.java

```diff
@@ -304,17 +297,24 @@ private boolean isFlexStop(
     }
 
     /**
-     * Calculate previous departure time, needed for all patterns. This is done by defining the 'last valid departure
+     * Calculate previous departure times, needed for all patterns. This is done by defining the 'last valid departure
      * time' for all stops except the first.
```

Comment:
Insert a comment about the assumption of what the previous departure time is for `stopSequence = 0`, and maybe repeat that comment in the test where the consecutive `2, 2` departure time sequence appears.

Reply:
I've updated to always return zero for the first stop which is much clearer.